src/app/settings/system-config/system-config.component.html

```diff
@@ -0,0 +1,58 @@
+<ul class="nav nav-tabs">
+    <li class="nav-item">
+        <button class="nav-link" [ngClass]="{'active': IsUsernameTabVisible}" id="username-tab" type="button"
+            (click)="LoadUsernameScreen()">Username</button>
+    </li>
+    <li class="nav-item">
+        <button class="nav-link" [ngClass]="{'active': IsPasswordTabVisible}" id="password-tab" type="button"
+            (click)="LoadPasswordScreen()">Password</button>
+    </li>
+
+    <li class="nav-item">
+        <button class="nav-link" [ngClass]="{'active': IsSessionTabVisible}" id="session-tab" type="button"
+            (click)="LoadSessionScreen()">Session</button>
+    </li>
+    <li class="nav-item">
+        <button class="nav-link" [ngClass]="{'active': IsSmtpTabVisible}" id="smpt-tab" type="button"
+            (click)="LoadSmtpScreen()">SMTP</button>
+    </li>
+
+    <li class="nav-item">
+        <button class="nav-link" [ngClass]="{'active': IsGlobalTabVisible}" id="global-tab" type="button"
+            (click)="LoadGlobalScreen()">Global</button>
+    </li>
+</ul>
+<div class="tab-content" *ngIf="isDataLoaded">
+    <div id="username" [hidden]="!IsUsernameTabVisible">
+
+        <app-username-config [usernameSettings]="usernameSettings">
+        </app-username-config>
+    </div>
+
+    <div id="password" [hidden]="!IsPasswordTabVisible">
```

Comment:
can we use ngIf instead of hidden. As with hidden all components will get loaded at once, even if we don't nedd them.